webpack.config.js

```diff
@@ -79,6 +79,7 @@ const nodeModulesToTranspile = [
 	'striptags/',
 	'unicode-match-property-ecmascript/',
 	'unicode-match-property-value-ecmascript/',
+	'gridicons/',
```

Comment:
Is the `gridicons/` transpilation related to the CoreJS 3 upgrade?

In any case, you need to also add `@automattic/calypso-polyfills` to the whitelist to make it all work.

Reply:
Possibly unrelated. This was a fix that was required for production builds to work locally for me, but it doesn't seem to be an issue outside of my machine. I'll remove that and double-check that everything still works fine.